sdk/core/core-http/rollup.config.ts

```diff
@@ -27,9 +27,11 @@ const nodeConfig = {
     "node-fetch",
     "os",
     "stream",
+    "tls",
     "tough-cookie",
     "tslib",
     "tunnel",
+    "util",
```

Comment:
This one is apparently needed because the `@azure/logger` library uses `util` and Rollup has started complaining about it.